.github/workflows/ci.yml

```diff
@@ -0,0 +1,91 @@
+name: Run Integration Tests
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    timeout-minutes: 60 
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
```

Comment:
Out of curiosity, how where these test assemblies picked? Is this at all related to the work that @JamesNK was going to do to gather stats on most common test failures?

Reply:
I plan to do all , but I just chose the worst ones from the ado stats